src/scss/mixins/_common.scss

```diff
@@ -23,71 +23,88 @@
 /// @access public
 /// @output Basic styles shared between input, textarea, select etc.
 @mixin oFormsCommonFieldBase {
-	@include oColorsFor(o-forms-field-standard);
-	@include oTypographySize($_o-forms-typography-scale-regular);
-	margin-top: $_o-forms-spacing;
-	box-sizing: border-box;
-	width: 100%;
-	max-width: $_o-forms-field-max-width;
-	min-height: $_o-forms-field-default-height;
-	padding: $_o-forms-field-default-padding-top $_o-forms-field-default-padding-leftright $_o-forms-field-default-padding-bottom;
-	border: $_o-forms-field-border;
-	border-radius: $_o-forms-field-border-radius;
-	background-clip: padding-box;
-	font-family: $o-forms-font-family;
-	outline: none;
-	transition: 0.15s box-shadow ease-in;
-	appearance: none;
+	@include oBrandConfigureFor('o-forms') {
+		@include oTypographySize($_o-forms-typography-scale-regular);
```

Comment:
Any particular reason are we using this mixin _and_ font-family instead of `oTypographySans(x)`?

Reply:
Unsure. Legacy? I'll update these.

Reply:
Oh actually, this is technically a breaking. As a nice to have for code consistency, I'll just create a breaking issue for it.